hide_windows.go

```diff
@@ -9,7 +9,12 @@ func hideFile(path string) error {
 	kernel32 := syscall.NewLazyDLL("kernel32.dll")
 	setFileAttributes := kernel32.NewProc("SetFileAttributesW")
 
-	r1, _, err := setFileAttributes.Call(uintptr(unsafe.Pointer(syscall.StringToUTF16Ptr(path))), 2)
+	utf16, err := syscall.UTF16PtrFromString(path)
+	if err != nil {
+		return err
+	}
+
+	r1, _, err := setFileAttributes.Call(uintptr(unsafe.Pointer(utf16)), 2)
```

Comment:
This change is not related to what is described in the PR description

I would have put in away from this PR and moved to another one, dedicated to this change.

Reply:
Oh, sorry about that. I accidentally overlooked it

Reply:
Thanks for confirming it.

I'll mark the PR as "changes required"